packages/compass-e2e-tests/tests/read-only.test.ts

```diff
@@ -0,0 +1,421 @@
+import { beforeTests, afterTests, afterTest } from '../helpers/compass';
+import { promises as fs } from 'fs';
+import path from 'path';
+import os from 'os';
+import { expect } from 'chai';
+import * as Selectors from '../helpers/selectors';
+import { createNumbersCollection } from '../helpers/insert-data';
+
+describe('readOnly: true / Read-Only Edition', function () {
+  let tmpdir: string;
+  let i = 0;
+
+  beforeEach(async function () {
+    tmpdir = path.join(
+      os.tmpdir(),
+      `compass-global-preferences-${Date.now().toString(32)}-${++i}`
+    );
+    await fs.mkdir(tmpdir, { recursive: true });
+  });
+
+  afterEach(async function () {
+    await fs.rmdir(tmpdir, { recursive: true });
+  });
+
+  it('hides and shows the plus icon on the siderbar to create a database', async function () {
+    const compass = await beforeTests();
+    try {
+      const browser = compass.browser;
+      await browser.setFeature('readOnly', true);
```

Comment:
Should this maybe be set back to false in the `finally` part?

Reply:
Yeah maybe it would be more explicit, but all tests start with setting it to false, but I can see your concern, will add.

Reply:
I’m more worried about this potentially affecting the next test that runs, if only this test + another test run, for example